app/grants/management/commands/contribution_checker.py

```diff
@@ -0,0 +1,51 @@
+import datetime as dt
+from time import sleep
+from datetime import timezone
```

Comment:
i think this script duplicates some of the stuff that the `update_tx_status` script does already...

Reply:
This script was just a one time use for me to find the suspicious transactions, not meant to be constantly running or anything like that. The frontend fixes should fix non-malicious behavior

Reply:
ok sounds good